README.md

```diff
@@ -43,10 +45,12 @@ $ php graph-composer.phar export ~/path/to/your/project
     ```bash
     $ php graph-composer.phar export ~/path/to/your/project export.png
     ```
-    
+
 *   You may optionally pass an `--format=[svg/svgz/png/jpeg/...]` option to set
     the image type (defaults to `svg`).
 
+*   By default, all dependecies is show. If you only want to see the production dependencies you can add the `--no-dev` option.
```

Comment:
How about "By default, graph-composer will show all dependencies (`require` and `require-dev` keywords). If you want to hide the `require-dev` dependencies, you can add the `--no-dev` option." ?

(see also above)